modules/nw-ovn-ipsec-enable.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc
+
+[id="nw-ovn-ipsec-enable_{context}"]
+= Enabling IPsec encryption
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* Use a cluster with the OVN-Kubernetes cluster network provider.
+
+.Procedure
+
+* To enable IPsec encryption, enter the following command:
++
+[source,terminal]
+----
+$ oc patch network.operator.openshift.io/cluster --type=merge -p \
+  '{
+    "spec": { "defaultNetwork" :{ "ovnKubernetesConfig" :{ "ipsecConfig" :{}}}}
+  }'
+----
```

Comment:
Beside To enable IPsec encryption, should we show how to disable IPsec encryption, such as: 
oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"ipsecConfig":null}}}}'

Reply:
@weliang1, sure.